pkg/frontend/transport/handler_test.go

```diff
@@ -178,7 +178,7 @@ func TestHandler_ServeHTTP(t *testing.T) {
 				require.Len(t, logger.logMessages, 1)
 
 				msg := logger.logMessages[0]
-				require.Len(t, msg, 18+len(tt.expectedParams))
+				require.Len(t, msg, 21+len(tt.expectedParams))
 				require.Equal(t, level.InfoValue(), msg["level"])
```

Comment:
Should we add a test case here that exercises the formatting for `start`, `end` and `step`?

Reply:
i added that in https://github.com/grafana/mimir/pull/6473/commits/8e6a8f4c6090fb2bea6b9088aa53d3b864cbb0ad